src/app/components/Toolbar/Features/AccountSelector/index.tsx

```diff
@@ -50,6 +51,10 @@ export const Account = memo((props: AccountProps) => {
     [WalletType.PrivateKey]: t('toolbar.wallets.type.privateKey', 'Private key'),
   }
 
+  const { details } = props
```

Comment:
Slight preference against extracting `details` into a variable (we still use `({props.details})` below, and we aren't extracting other props)

Reply:
OK, not extracting in any more. (In this function.)